src/Symfony/Component/Dotenv/Command/DebugCommand.php

```diff
@@ -50,7 +50,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             return 1;
         }
 
-        $filePath = $this->projectDirectory.\DIRECTORY_SEPARATOR.'.env';
+        $composerFile = $this->projectDirectory . '/composer.json';
+        $config = (is_file($composerFile) ? json_decode(file_get_contents($composerFile), true) : [])['extra']['runtime'] ?? [];
+        $filePath = $this->projectDirectory.\DIRECTORY_SEPARATOR.($config['dotenv_path'] ?? '.env');
```

Comment:
```suggestion
        $filePath = $this->projectDirectory.'/'.($config['dotenv_path'] ?? '.env');
```

using the constant is not necessary, let's replace it when we are at it